### ISSUE TITLE
Ctrl+Enter to save and refresh

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#764]
As an alternative to clicking elsewhere, I suggest to activate also Ctrl+Enter as a combination which will ellicit save and reload.

**Comment by davidzbiral** [comment]
@Ptrhnk Perhaps implemented? Close if so.

**Comment by Ptrhnk** [comment]
@davidzbiral Nope, It maybe looks like it works because pressing ctrl will not disable enter. Problem is when combination of things happens. (I've recorded use case for testing)

https://user-images.githubusercontent.com/29732281/163801600-d593be71-132d-401c-8b41-b630f591b895.mov

**Comment by davidzbiral** [comment]
I think this is not a priority anymore, based on real-life testing. Closing.
